src/main/java/com/capitalone/dashboard/controller/GitHubController.java

```diff
@@ -69,4 +72,9 @@ public ResponseEntity<String> refresh(@Valid String url) throws HygieiaException
     public ResponseEntity<String> cleanup() throws HygieiaException {
         return gitHubService.cleanup();
     }
+
+    @RequestMapping(value = "/syncPullRequest", consumes = APPLICATION_JSON_VALUE,method = POST, produces = APPLICATION_JSON_VALUE)
```

Comment:
Do we still need a similar endpoint from api project? can it be removed from api as its a tool specific project